modules/core/src/main/scala/org/scalasteward/core/util/dateTime.scala

```diff
@@ -31,19 +31,25 @@ object dateTime {
   def renderFiniteDuration(fd: FiniteDuration): String =
     fd.toString.filterNot(_.isSpaceChar)
 
-  def showDuration(d: FiniteDuration): String = {
-    def symbol(unit: TimeUnit): String =
-      unit match {
-        case DAYS         => "d"
-        case HOURS        => "h"
-        case MINUTES      => "m"
-        case SECONDS      => "s"
-        case MILLISECONDS => "ms"
-        case MICROSECONDS => "µs"
-        case NANOSECONDS  => "ns"
-      }
-    splitDuration(d).map(d1 => d1.length.toString + symbol(d1.unit)).mkString(" ")
-  }
+  private def symbol(unit: TimeUnit): String =
+    unit match {
+      case DAYS         => "d"
+      case HOURS        => "h"
+      case MINUTES      => "m"
+      case SECONDS      => "s"
+      case MILLISECONDS => "ms"
+      case MICROSECONDS => "µs"
+      case NANOSECONDS  => "ns"
+    }
+
+  private def showDurationList(d: FiniteDuration): List[String] =
+    splitDuration(d).map(d1 => d1.length.toString + symbol(d1.unit))
+
+  def showDuration(d: FiniteDuration): String =
+    showDurationList(d).mkString(" ")
+
+  def showDurationShort(d: FiniteDuration): String =
+    showDurationList(d).take(3).mkString(" ")
```

Comment:
I like this change. When we show the duration, there is probably never the need to be as precise as we are now. I would just change the old `showDuration` to only use the first three elements so that we don't need a new `showDurationShort`.

Oh, and it would be nice to have this change in a separate PR, since this feature is independent of the change in `showDuration` and the other PR could be merged right away.

Reply:
👍 
Extracted PR https://github.com/scala-steward-org/scala-steward/pull/2517